test_crates/enum_repr_int_changed/old/src/lib.rs

```diff
@@ -22,30 +22,85 @@ pub enum IsizeToUsizeEnum {
     Baz,
 }
 
-// The following enums have *removals* of repr(i*) and repr(u*),
-// not changes to another repr(i*) or repr(u*).
-// They should not be reported by this rule, because they have their own rule.
+#[repr(u8, C)]
+pub enum U8CToU16CEnum {
+    Bar,
+    Baz,
+}
+
+#[repr(C, u8)]
+pub enum CU8ToCU16Enum {
+    Bar,
+    Baz,
+}
+
+#[repr(u8, C)]
+pub enum U8CToSeparateU16CEnum {
+    Bar,
+    Baz,
+}
+
+
+// The following enums have *rearrangements* of repr(*), potentially
+// splitting singular repr(*) into multiple, smaller repr(*) or merging
+// repr(*) into larger repr(*).
+// They should not be reported by this rule, because they are legal.
```

Comment:
Can you add this matching comment to the `new` version of this crate?

In general, we want the `old` and `new` crates' source code to be structurally as close as possible to each other, so that one can read them side-by-side and always know which items correspond to each other. Comment blocks like this help break up the long sequences of nothing-but-code, and act as visual anchors when doing that kind of side-by-side reading of the two crates.

Reply:
I added the missing comment, indeed it's a lot easier to navigate in the two test files having matching comments in both of them.